src/Dialogue/Dialogue.tsx

```diff
@@ -10,17 +10,17 @@ const TimesIcon: JSX.Element = (
 export interface DialogueProps {
     className?: string;
     desc?: string | JSX.Element | React.ReactNode;
-    disablePrimaryBtn?: boolean;
-    disableSecondaryBtn?: boolean;
     enableBackdropDismiss?: boolean;
     enableCloseButton?: boolean;
     header?: string | JSX.Element | React.ReactNode;
     id?: string;
     onDismiss?: (e?: React.MouseEvent<HTMLDivElement>) => void;
     primaryAction?: (e?: React.MouseEvent<HTMLButtonElement>) => void;
     primaryBtn?: string | JSX.Element;
+    primaryBtnDisabled?: boolean;
```

Comment:
why is it in the past tense disablePrimaryBtn looks better

Reply:
it used to be `disablePrimaryBtn` but this has been changed a while back when @mario-subo indicated that it sounds like an action instead of a state.